zhaquirks/linxura/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Linxura button device."""
+
+from zigpy.quirks import CustomCluster
+import zigpy.zcl.clusters.security
```

Comment:
Just import IasZone here: `from zigpy.zcl.clusters.security import IasZone`.

```suggestion
import zigpy.zcl.clusters.security
from zigpy.zcl.clusters.security import IasZone
```